src/autofeat/featsel.py

```diff
@@ -43,6 +43,7 @@ def _noise_filtering(
     target: np.ndarray,
     good_cols: list | None = None,
     problem_type: str = "regression",
+    random_seed: int = None
```

Comment:
I think mypy would also complain here since the typing should be `int | None = None`

I will include a guide for how to contribute to this repo and maybe also add some pipelines to run automated checks on github, but for now please make sure you have poetry installed, then install the packages with
`poetry install`
and then run the automated formatting and checks (using ruff and others) with
`poetry run poe format` and `poetry run poe check` to make sure all these easy to fix issues are addressed. Thanks! :)

Reply:
Well, I managed to install all packages using poetry. However I cannot get anything meaningful running poetry format checks (need to spend with it more time later). Anyway I updated typing hints as you proposed.